### DISCUSSION TITLE
Suggestion: block city expension

### DISCUSSION THREAD

**Discussion by Thiche** [voxeltycoon/issues#533] [General]
Add an mean to block city expension on undesired areas with e.g. fences (like the mod but available in the genuine game) or by fixing a zone to "industry type" where city cannot expend.

**Comment by victorleblais** [comment]
I'm sure most people know, but OpenTTD has a similar feature: Buying land.

As you said, you can do a similar thing by placing decorations (e.g. pumpkins) in front of the roads you want to stop expanding.
